src/pages/HomePage.js

```diff
@@ -50,7 +50,7 @@ function HomePage() {
 
     return (
         <><center>
-            <div id="top"></div>
+            <div id="top"/>
 
             <br></br>

```

Comment:
Great Job with the logo! It really improves the appearance of our overall project.

Reply:
I'm glad we came up with the idea for a logo; definitely makes it look more professional